lemur/certificates/cli.py

```diff
@@ -775,3 +777,146 @@ def deactivate_entrust_certificates():
             current_app.logger.info(log_data)
             sentry.captureException()
             current_app.logger.exception(e)
+
+
+@manager.option("-c", "--commit", dest="commit", action="store_true", default=False, help="Persist changes.")
+def disable_rotation_of_duplicate_certificates(commit):
+    log_data = {
+        "function": f"{__name__}.{sys._getframe().f_code.co_name}",
+        "message": "Disabling auto-rotate for duplicate certificates"
+    }
+
+    if commit:
+        print("[!] Running in COMMIT mode.")
+
+    authority_names = current_app.config.get("AUTHORITY_TO_DISABLE_ROTATE_OF_DUPLICATE_CERTIFICATES")
+    if not authority_names:
+        log_data["message"] = "Skipping task: No authorities configured"
+        current_app.logger.debug(log_data)
+        return
+
+    log_data["authorities"] = authority_names
+
+    authority_ids = [a.id for a in Authority.query.filter(Authority.name.in_(authority_names)).all()]
+    duplicate_candidate_certs = list_duplicate_certs_by_authority(authority_ids)
+
+    log_data["certs_with_serial_number_count"] = len(duplicate_candidate_certs)
+    current_app.logger.info(log_data)
+
+    skipped_certs = []
+    rotation_disabled_certs = []
+    unique_prefix = []
+    failed_certs = []
+
+    for duplicate_candidate_cert in duplicate_candidate_certs:
+        success, duplicates = process_duplicates(duplicate_candidate_cert,
```

Comment:
Since boolean, I've used `success` and then further check is `if not success:`
For `status`, I feel appropriate value will be `success/failure` and further check will be string comparison. I prefer boolean over that. Let me know more if I missed the thought behind this.